# Module enum
int8 module

bool clamp

bool turn_over
